source/fundamentals/builders/aggregates.txt

```diff
@@ -50,6 +48,32 @@ aggregation as a list:
    :language: java
    :dedent:
 
+Documents
+---------
+
+Use the ``documents()`` method to create a
+:manual:`$documents </reference/operator/aggregation/documents/>`
+pipeline stage that returns literal documents from query values.
+
+.. important::
+
+   If you use the ``$documents`` stage in an aggregation pipeline, it must be the first 
+   stage in the pipeline.
+
+The following example creates a pipeline stage that creates 
+sample documents with a ``"title"`` field: 
```

Comment:
I: elsewhere on the page, field names are not in quotes

S:
```suggestion
sample documents with a ``title`` field: 
```

Reply:
Good catch!